src/Network/Network/Common/NetworkResourceManagerProfile.cs

```diff
@@ -2305,6 +2305,16 @@ private static void Initialize()
                 cfg.CreateMap<ANM.PSNetworkManagerSecurityAdminRuleCollection, MNM.AdminRuleCollection>();
                 cfg.CreateMap<ANM.PSNetworkManagerScopeConnection, MNM.ScopeConnection>();
                 cfg.CreateMap<ANM.PSNetworkManagerConnection, MNM.NetworkManagerConnection>();
+                cfg.CreateMap<ANM.PSNetworkManagerRoutingRule, MNM.RoutingRule>();
```

Comment:
in MNM the sdk or powershell? I think we should keep the 'networkmanager' prefix on the powershell and sdk objects to avoid conflicts

if MNM is the sdk object, then I'm a bit confused because I thought we prefixed everything with 'networkmanager', ie 'networkmanagerroutingrule'

Reply:
MNM is SDK. We only prefixed the routing configuration with NetworkManager prefix.

cfg.CreateMap<ANM.PSNetworkManagerRoutingConfiguration, MNM.NetworkManagerRoutingConfiguration>();
cfg.CreateMap<ANM.PSNetworkManagerRoutingGroupItem, MNM.NetworkManagerRoutingGroupItem>();
cfg.CreateMap<ANM.PSNetworkManagerRoutingRule, MNM.RoutingRule>();
cfg.CreateMap<ANM.PSNetworkManagerRoutingRuleCollection, MNM.RoutingRuleCollection>();

Reply:
sounds good, thanks for clarifying